shell/types/menu-actions.ts

```diff
@@ -0,0 +1,11 @@
+// NB: for some reason, this is not being picked up by the linter as being used.
+export enum MenuActions {
+    Pause = 'Pause', // eslint-disable-line no-unused-vars
```

Comment:
Sorry my mistake, this is obviously fine if used for E2E. 
We should probably move it within the E2E area to avoid confusion and keep it outside the code.